addons/website_quote/views/website_quotation_backend.xml

```diff
@@ -8,10 +8,11 @@
             <field name="arch" type="xml">
                 <xpath expr="//header/button[@name='action_button_confirm']" position="before">
                     <button name="open_quotation" string="Preview Quotation" type="object" class="oe_highlight"
-                        attrs="{'invisible': ['|',('template_id','=',False),('state','=','draft')]}"/>
+                        attrs="{'invisible': ['|','&amp;',('template_id','=',False),('state','=','draft'),('quote_viewed','=',True)]}"/>
                 </xpath>
                 <xpath expr="//header/button[@name='action_button_confirm']" position="before">
-                    <button name="open_quotation" string="Preview Quotation" type="object" attrs="{'invisible': ['|',('template_id','=',False),('state','!=','draft')]}"/>
+                    <button name="open_quotation" string="Preview Quotation" type="object" 
+                        attrs="{'invisible': ['|','&amp;',('template_id','=',False),('state','!=','draft'),('quote_viewed','=',False)]}"/>
```

Comment:
This button `open_quotation` is present twice. One is enough :)
The domain `'invisible': [('template_id','=',False)]` is enough (don't care about the state).
The `quote_viewed` is for the `oe_highlight` class, not for the invisible attribute.